src/components/Integrations/Shopify/Shopify.js

```diff
@@ -118,9 +118,12 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
                 </div>
               </div>
               <footer className="dp-integration__actions">
-                <button type="button" className="dp-button button-medium primary-grey">
+                <a
+                  href={_('common.control_panel_url')}
+                  className="dp-button button-medium primary-grey"
+                >
                   {_('common.back')}
-                </button>
+                </a>
```

Comment:
Is it not possible to avoid coding duplication for this button?

Reply:
I can add a fragment for this, yes 😄

Reply:
Fixed